mobile/README.md

```diff
@@ -38,6 +54,29 @@ To checkout the latest tagged release of lnd, run
 ⛰  git checkout $(git describe --tags --abbrev=0)
 ```
 
+Or, the second option:
+
+- You have to be at `$GOPATH/src/github.com/lightningnetwork/` directory.
```

Comment:
With the dot in the clone command, you need to be in `$GOPATH/src/github.com/lightningnetwork/lnd/`. The dot means "do not create a directory with the same name as the repository, put everything in the current folder". So if you have this first sentence then the dot isn't needed.